README.md

```diff
@@ -31,126 +31,50 @@ Advantages:
 
 Maestro communicates to Pelion Cloud over https outbouund. It stores its config locally in a private database, but can also use DeviceDB for storage of applications, network settings, configs and other data when used in conjuction with standard Pelion Cloud services.
 
-### Interacting
-
 If you are locally on a gateway / edge system using maestro, you should explore [maestro-shell](https://github.com/armPelionEdge/maestro-shell) which will let you interact with maestro directly using the local API.
 
-### Building
-
-#### prerequisites
-
-Build pre-requisites:
-
-* golang
-* python
-* gcc
-
-Maestro has a simple internal preprocessor `build.sh` script which use `m4` - make sure `m4` is installed on your build system (normally included with gcc)
-
-You will need gcc for compiling maestro's dependencies.
-
-*On Ubuntu*
-
-`sudo apt-get install build-essential python`  should take care of all this.
-
-If you go not have a Go build environment up, you will need to install [golang](https://golang.org/dl/). Just expand the tar ball and put in in /opt, with `sudo mv go /opt`
-
-You also will need your golang env vars setup correctly. Here is a script you can just run, assuming your Go workspace is at `$HOME/work/gostuff`:
-
-*setup-go.sh*
-```
-#!/bin/bash
-
-export GIT_TERMINAL_PROMPT=1
-export GOROOT=/opt/go
-export GOPATH="$HOME/work/gostuff"
-export GOBIN="$HOME/work/gostuff/bin"
-export PATH="$PATH:$GOROOT/bin:$GOBIN"
-```
-
-then just run: `source ./setup-go.sh` 
-
-Run your build commands from this sub-shell.
-
-##### On Arch
-
- * install go:
-  `sudo pacman -S golang`
+## Pre-Requisites
 
- * Add generated executables into your path (so that you can do stuff like `maestro` on the command line)
-   ```
-   export PATH="$PATH:$HOME/go/bin"
-   ```
+Install the following tools:
 
+* [vagrant](https://www.vagrantup.com/) (version 2.2.6)
+* [virtualbox](https://www.virtualbox.org/) (version 6.0)
 
-#### build instructions
+Install Vagrant plugins:
 
-`go get github.com/armPelionEdge/maestro`
-
-Enter github credentials as needed. Now, where `$GOPATH` is your go workspace folder, as in the sub-shell above...
-
-```
-cd $GOPATH/src/github.com/armPelionEdge/maestro
+```bash
+vagrant plugin install vagrant-reload
```

Comment:
`vagrant-reload` is a plugin that allows vagrant to reboot a VM during provisioning.

We need this because we change network adapter names from the new style (Ubuntu 16.04 and above: `enp0s3`, `enp03as4`, etc.) to the conventional style (Pre 16.04 - `eth0`, `eth1`, etc). This requires a reboot.